pkg/projects/Microsoft.NETCore.App/netcoreapp1.1/project.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
What are the package differences between netcoreapp1.0 and the netcoreapp1.1 package list?